src/preferences/dialog/dlgprefmixer.cpp

```diff
@@ -72,13 +75,14 @@ DlgPrefMixer::DlgPrefMixer(
         : DlgPreferencePage(pParent),
           m_pConfig(pConfig),
           m_xFaderMode(MIXXX_XFADER_ADDITIVE),
-          m_transform(EngineXfader::kTransformDefault),
-          m_cal(0.0),
-          m_mode(kXfaderModeKey),
-          m_curve(kXfaderCurveKey),
-          m_calibration(kXfaderCalibrationKey),
-          m_reverse(kXfaderReverseKey),
-          m_crossfader("[Master]", "crossfader"),
+          m_xFaderTransform(EngineXfader::kTransformDefault),
+          m_xFaderCal(0.0),
```

Comment:
proposal: Create a `struct CrossfaderParameter` in the class to bundle the calibration, transform, reverse and mode together instead of prefixing all of those with `xFader`?

Reply:
You mean just for holding the parameters in the preferences?
Sure, we can do that. But tbh I'd like to get the mechanics right first before we start optimizing it.

Reply:
Yup, let's postpone it.
Wanna file a request for it?

Reply:
nah, its no need to file an issue over code improvement. someone else will clean it up when they look at it again.